orchestrator/gravity_utils/Cargo.toml

```diff
@@ -11,6 +11,7 @@ gravity_abi = { path = "../gravity_abi" }
 gravity_proto = { path = "../gravity_proto/" }
 cosmos-sdk-proto = "0.6.3"
 deep_space = { git = "https://github.com/iqlusioninc/deep_space/", branch = "master" }
+eth_checksum = "0.1.2"
```

Comment:
Will it be a good idea to vendor in the `eth_checksum` code in the repo? It's only a single function ([here](https://github.com/miguelmota/rust-eth-checksum/blob/master/src/lib.rs)). Will reduce one dependency.

Reply:
And we already have `sha3` as a dependency.

Reply:
good call, I have removed the dependency